src/Command/ModelCommand.php

```diff
@@ -1546,4 +1554,43 @@ protected function bakeEnums(Table $model, array $data, Arguments $args, Console
             $enumCommand->execute($args, $io);
         }
     }
+
+    /**
+     * @param array<string, array<string, mixed>> $associations
+     * @return array<string, array<string, mixed>>
+     */
+    protected function ensureAliasUniqueness(array $associations): array
```

Comment:
```suggestion
    protected function ensureUniqueAlias(array $associations): array
```
This rolls a bit easier off the tongue :)

Reply:
But it does it for more than one, so we could debate `ensureUniqueAliases()` etc.

Reply:
I must say I kinda like the naming ensureAliasUniqueness() that @markstory proposed.

Reply:
Okay, we can keep the same then.

Reply:
Oh createAssociationAlias() was from him, not this one.
Either way, if all are fine, we can keep it.